### DISCUSSION TITLE
Installation via package managers

### DISCUSSION THREAD

**Discussion by edeandrea** [Hyperfoil/Hyperfoil#297] [Ideas]
Hello! I'm excited to be here!

I was following along with [the quickstart](https://hyperfoil.io/quickstart/quickstart1.html), which mentions using `wget` to download/install hyperfoil.

It would be nice to be able to install (& update/upgrade) via a package manager (homebrew/snap/dnf/sdkman/apt/jbang/etc).

The fact, though, that I need to `cd` into the installation directory (i.e. `hyperfoil-0.24.2`) and run `bin/cli.sh` from there does seem to make it "unfriendly" to package managers.

**Comment by jesperpedersen** [comment]
Something that you are willing to work on ?

**Reply by edeandrea** [reply]
Me? Probably not. I'm just a user. Don't know much about Hyperfoil. Just getting started with it and going to build out some benchmarks for https://github.com/quarkusio/quarkus-super-heroes

**Reply by jesperpedersen** [reply]
Gotcha !

**Reply by edeandrea** [reply]
I mean, I can try in the future but at this moment in time I don't have the bandwidth I can't commit to doing it.
